oc_mirror.sh

```diff
@@ -49,11 +49,8 @@ function setup_quay_mirror_registry() {
    popd
 }
 
-# Set up a mirror using the 'oc mirror' command
-# The backend registry can be either 'podman' or 'quay'
-function setup_oc_mirror() {
-
-   update_docker_config
+# Mirror the upstream channel directly to the local registry
+function mirror_to_mirror_publish() {
```

Comment:
Probably in future it could be interesting also to mirror OLM components

Reply:
Yes I think we can easily do that here in a separate commit.